src/interface/variable.cpp

```diff
@@ -113,6 +113,12 @@ void CellVariable<T>::allocateComms(std::weak_ptr<MeshBlock> wpmb) {
 
   // enroll CellCenteredBoundaryVariable object
   vbvar->bvar_index = pmb->pbval->bvars.size();
+  // TODO(JMM): This means RestrictBoundaries()
```

Comment:
Is the implication that we'll currently be launching  something like 2x more kernels than necessary for dealing with boundaries?

Reply:
we're launching `nstages x` more kernels than necessary for dealing with boundaries. I didn't fix that here, because it may be larger surgery. But it should be fixed. I can create an issue.

Reply:
#366